### PR TITLE
Farsight DNSDB lookup integration

### PR DESCRIPTION
**Pull request by gshriki**
@liorkol - lets review and decide on context setting if needed

### CODE REVIEW

File: Integrations/integration-DNSDB.yml [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                }
+            }
+            md += '\n';
+        }
+        return md;
+    }
+
+    var sendRequest = function(requestUrl,parameters) {
+        var url = serverUrl + requestUrl + encodeToURLQuery(parameters);
+        console.log(url);
```
Comment (slavikm) [inline Integrations/integration-DNSDB.yml]: Please remove logs

File: Integrations/integration-DNSDB.yml [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+        var res = http(
```
Comment (slavikm) [inline Integrations/integration-DNSDB.yml]: We're trying to make it a standard to at least add the proxy parameter to all integrations

File: Integrations/integration-DNSDB.yml [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+            }
+        );
+        if (res.StatusCode < 200 || res.StatusCode>299) {
+            throw 'Error ' + res.StatusCode + '. ' + res.Status;
+        }
+        var obj={};
+        obj.entries=[];
+        var array = res.Body.split('\n');
+        for (var i=0;i<array.length;i++) {
+            console.log('parsing ' + array[i]);
```
Comment (slavikm) [inline Integrations/integration-DNSDB.yml]: logs

File: Integrations/integration-DNSDB.yml [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+            if (array[i] && array[i].length > 0) {
+                try {
+                    obj.entries.push(JSON.parse(array[i]));
+                } catch (err) {
+
```
Comment (slavikm) [inline Integrations/integration-DNSDB.yml]: empty?

**Comment by meirwah** [comment]
@gshriki you want to merge this?

**Comment by gshriki** [comment]
@liorkol said he will review first, and then merge...

**Pull request merged**